src/views/InvitesManage.js

```diff
@@ -140,44 +139,42 @@ class InvitesManage extends React.Component {
     });
 
     return (
-      <Row>
-        <Col>
-          <p>
-            {
-              'manage invites here, for stars. can only give invites to child planets'
-            }
-          </p>
-
-          {spawnProxyWarning}
-
-          <PointInput
-            prop-format="innerlabel"
-            prop-size="lg"
-            placeholder={'~sampel-sipnem'}
-            value={this.state.planetInput}
-            onChange={this.handlePointInput}>
-            <InnerLabel>{'Planet to set invites for'}</InnerLabel>
-            <ValidatedSigil
-              patp={this.state.targetPlanet}
-              validator={() => this.validatePoint}
-            />
-          </PointInput>
-
-          <Input
-            prop-format="innerlabel"
-            prop-size="lg"
-            value={this.state.targetPoolSize}
-            onChange={this.handlePoolSizeInput}>
-            <InnerLabel>{`Available invites ${poolSizeText}`}</InnerLabel>
-          </Input>
-
-          <StatelessTransaction
-            canGenerate={this.state.canGenerate}
-            createUnsignedTxn={this.createUnsignedTxn}
-            ref={this.statelessRef}
+      <View>
+        <p>
+          {
+            'manage invites here, for stars. can only give invites to child planets'
```

Comment:
Just a heads up, this is description is outdated. Stars can now grant invites to anyone.

Reply:
cool, will file that away for a copy pass 👍

Reply:
I wouldn't worry too much about copy if I were you! Your time is better spent elsewhere. (: